project_templates/conda_exercises/exercise_2/fleeqtk_seq.xml

```diff
@@ -1,4 +1,7 @@
 <tool id="fleeqtk_seq" name="Convert to FASTA (fleeqtk)" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="1.3">fleeqtk</requirement>
+    </requirements>
```

Comment:
But this is part of the exercise right? Maybe make that more clear in the exercise written description?

Reply:
It is not mentioned anywhere in https://planemo.readthedocs.io/en/latest/writing_advanced.html#exercise-build-a-recipe , it's more of a surprise egg...
How to add a requirement was already explained for `seqtk_seq` using `planemo tool_init` in https://planemo.readthedocs.io/en/latest/writing_standalone.html#the-basics, it is not clear at all why it's removed here for `fleeqtk_seq`, unless the goal is to confuse people.